### ISSUE TITLE
terraform-google-modules dependency tree reverse engineering

### ISSUE THREAD

**Issue by obriensystems** [terraform-google-modules/terraform-example-foundation#1147]
talking to Marian.
LZ customization on demand - script to modify tfvars - one option

### TL;DR

local ../
and remote modules
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/2-environments/modules/env_baseline/monitoring.tf#L22

```
module "monitoring_project" {
  source  = "terraform-google-modules/project-factory/google"
  version = "~> 14.0"
```
see
https://registry.terraform.io/modules/terraform-google-modules/project-factory/google/14.4.0

some modules are from GoogleCloudPlatform
```
module "ilbs" {
  source   = "GoogleCloudPlatform/lb-internal/google"
  version  = "~> 5.0"
  for_each = toset(var.regions)
```
see
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/3-networks-hub-and-spoke/modules/transitivity/main.tf#L92

Start with the terraform plugin in Jetforms intelliJ
<img width="978" alt="Screenshot 2024-03-08 at 07 23 06" src="https://github.com/terraform-google-modules/terraform-example-foundation/assets/24765473/1349b0f6-1497-4ade-9d84-fadbd904b2d8">

assigned to fmichaelobrien

**Comment by fmichaelobrien** [comment]
bump for 23day stale state
